src/main/java/ca/uhn/fhir/jpa/starter/common/StarterJpaConfig.java

```diff
@@ -1,5 +1,37 @@
 package ca.uhn.fhir.jpa.starter.common;
 
+import static ca.uhn.fhir.jpa.starter.common.validation.IRepositoryValidationInterceptorFactory.ENABLE_REPOSITORY_VALIDATING_INTERCEPTOR;
```

Comment:
Where did this whole block come from? I doesn't appear in the upstream repo.

Reply:
This file appears to be very different than the upstream one. The only significant change we've made in our fork is to register interceptors when OAuth is enabled (lines ~ 443-449).